sp/src/game/server/hl2/npc_combine.cpp

```diff
@@ -4418,7 +4526,25 @@ void CNPC_Combine::HandleAnimEvent( animevent_t *pEvent )
 
 						EmitSound( "NPC_Combine.WeaponBash" );
 					}
-				}			
+				}	
+#ifdef EZ2
+				else
+				{
+					// Apply a velocity to hit entity if it has a physics movetype
+					if ( pHurt && pHurt->GetMoveType() == MOVETYPE_VPHYSICS )
```

Comment:
Should we also dispatch a kick interaction? Or is that not a good idea?

Reply:
I considered this, but I want to just follow through with this as it is. We can see what dispatching a kick interaction might look like if we use `ACT_MELEE_ATTACK2` on soldiers further, e.g. for attacking headcrabs or baby bullsquids.